cgroup2/manager.go

```diff
@@ -541,14 +535,6 @@ func (c *Manager) Stat() (*stats.Metrics, error) {
 			}
 		}
 	}
-	for _, name := range singleValueFiles {
-		if err := readSingleFile(c.path, name, out); err != nil {
-			if os.IsNotExist(err) {
-				continue
-			}
-			return nil, err
-		}
-	}
```

Comment:
I guess there is a slight distinction in that the old behavior, if we failed to read the file and it wasn't because it didn't exist we'd error out of Stat(), but.. not sure that's worth keeping around like 3 of the same function

Reply:
Yes, I saw that one and did not think it's a deal breaker ever.

Reply:
I don't either (you can resolve this, just wanted to point out the only behavioral difference I could see in the first commit)